packages/ui/src/views/platform/entity-reference-list.tsx

```diff
@@ -44,7 +42,6 @@ export function EntityReferenceList<T extends BaseEntityProps, S = string, F = s
   defaultEntityRenderer,
   parentFolderRenderer,
   childFolderRenderer,
-  apiError,
```

Comment:
This prop was never actually passed, and was handled the wrong way.
API error for this component is handled properly in PlatformUI package.